extensions.json

```
@@ -103,8 +103,8 @@
     {
       "id": "arcanis.vscode-zipfs",
       "repository": "https://github.com/yarnpkg/berry",
-      "version": "2.2.4",
-      "checkout": "vscode-zipfs/2.2.4",
+      "version": "2.3.0-rc.5",
+      "checkout": "vscode-zipfs/2.3.0-rc.5",
```

False positive (`-rc`):

```suggestion
      "version": "2.2.4",
      "checkout": "vscode-zipfs/2.2.4",
```